VAMobile/documentation/design/About/content-writers.md

```diff
@@ -0,0 +1,6 @@
+---
+title: For content writers
+sidebar_position: 3
+---
+
+Content development is in progress and will be published by the end of Q3.
```

Comment:
Can we add draft: true to the front matter on empty pages like this and make sure they aren't in the nav until they're ready? https://docusaurus.io/docs/next/api/plugins/@docusaurus/plugin-content-pages